presto-kafka/src/main/java/com/facebook/presto/kafka/KafkaSplit.java

```diff
@@ -160,8 +186,8 @@ public String toString()
                 .add("keyDataSchemaContents", keyDataSchemaContents)
                 .add("messageDataSchemaContents", messageDataSchemaContents)
                 .add("partitionId", partitionId)
-                .add("start", start)
```

Comment:
we still need start end for toString, right?

Reply:
Both start and end are initialized with default value of `0` when splits are built. 
They are set with right values after consumer talking to partitions on Presto workers through API getOffsetsByTimeStamp(). 

To my understanding, toString is used for serialization and Deserialization when splits are passed among workers. Since setting the two variables happens locally on a Presto worker. We don't need them in toString. 

But I think you are right to keep them for clarity and simplicity. Added them back.